Standards/scs-xxxx-v1-networking-standard.md

```diff
@@ -0,0 +1,88 @@
+External networks, on the other hand, are required to make VMs in a project externally accessible, which is an essential functionality, and as such should be mandatory in an SCS compliant cloud.
+
+Connecting a VM to a non-shared external network does at minimum require the creation of a virtual router and a project-internal network and subnet.
+Assigning public IPs to VMs in an internal network requires an internal subnet with public IP allocation ranges, that will usually be allocated from a CSP-managed subnet pool.
+To allow users the dynamic allocation and deallocation of publicly routable subnets, CSPs need to provide [dynamic routing via BGP](https://docs.openstack.org/neutron/latest/admin/config-bgp-dynamic-routing.html).
+
+Alternatively, at least for IPv4, virtual routers can also act as source NAT, allowing VMs in the internal network to access the internet through the routers external gateway IP.
+To make VMs in a NATed subnet externally accessible, OpenStack also supports destination NAT in the form of Floating IPs.
+A floating IP represents an address from the external network which can be allocated from a CSP-managed pool and mapped onto an IP within an internal network.
+
+NAT and floating IPs do not require dynamic routing and can greatly reduce the number of required IPv4 addresses, which are an increasingly sparse resource and may be subject to tight project-quotas by the CSP.
+Being ultimately a workaround for the limited IPv4 address space, neither are available for IPv6.
+It is possible, however, to create a dual-stack setup in which VMs attached to a routed internal network can have both public IPv6 addresses and private, NATed IPv4 addresses.
+
+[...]
+
+A shared external network may provide some usability benefits over a purely external one, as users can make their VMs directly accessible from the internet without creating additional resources.
+Beyond that, skipping virtual routers may be important for projects implementing some form of network function virtualisation (NFV) within a VM, but this is more of a niche application, that may also require disabling port security.
+
+[...]
+
+## Decision
+### API Extensions
+
+[...]
+
+### Policies
+
+[...]
+
+### Provider Networks
+
+CSPs **MUST** provide a standard external network that gives projects access to the internet.
+
+The external network **MUST** have an IPv6 subnet.
+CSPs **MUST** provide a subnet pool for the allocation of at least one public /64 IPv6 prefix per project.
+
+The external network **SHOULD** have an IPv4 subnet.
+If CSPs provide an IPv4 subnet, then CSPs **MUST** at least one public Floating IP per project.
+They **SHOULD** also provide a subnet pool for the allocation of IPv4 prefixes to project networks.
+
+CSPs **MUST** provide dynamic routing for all project-allocated IP-prefixes.
+
+All subnets of external networks **MUST** be configured with `--no-dhcp`.
```

Comment:
This is probably only relevant for shared external networks, which allow direct attachment of VMs